docs/AllelicCapSeg/ACS-methods.tex

```diff
@@ -109,7 +109,7 @@ \subsection{Segmented model} \label{segmented-model}
 \begin{equation}
 P(z_{ja(r,o)} = 1 | {\bf \pi}) = \pi_{a(r,o)}
 \end{equation}
-We set the alt and ref minor probabilities equal so that the only free parameter is $\pi_o$, with $\pi_{a(r)} = (1 - \pi_o)/2$.
+We set the alt and ref minor probabilities equal so that the only free parameter is $\pi = \pi_o$, with $\pi_{a(r)} = (1 - \pi)/2$.
 %
 The Bayesian network corresponding to this model is shown in Figure \ref{graphical_model}.
```

Comment:
@samuelklee It's a Python package called DAFT.  It's quite simple and I'm not sure it's expressive enough to handle some of the HDP-HMM weirdness that we may some day resort to, but it's very user-friendly.